pkgs/tools/security/ipscan/default.nix

```
@@ -0,0 +1,38 @@
+{ stdenv, fetchurl, jdk, jre, swt, makeWrapper, xorg }:
+
+let
+  pname = "ipscan";
+  version = "3.6.2";
+
+  jar = fetchurl {
+    url = "https://github.com/angryip/ipscan/releases/download/${version}/ipscan-linux64-${version}.jar";
+    sha256 = "0fmrrxanw46g53jvlb50516jqwqjzky564bf3df9i3ymjffrfnw4";
+  };
+
+  jarName = "${pname}-${version}.jar";
+  jarPath = "$out/share/java/${jarName}";
+in stdenv.mkDerivation rec {
+  inherit pname version;
+
+  dontUnpack = true;
+
+  buildInputs = [ makeWrapper jdk ];
+
+  installPhase = ''
+    mkdir -p $out/share/java
+    ln -s ${jar} ${jarPath}
```

```suggestion
    cp $src $out/share/${pname}.jar
```

as per `doc/languages-frameworks/java.xml`, this is not something imported by other java packages, right?